src/ResourceFactory.php

```diff
@@ -7,8 +7,8 @@
 namespace ZF\Hal;
 
 use Closure;
-use ZF\Hal\Collection;
-use ZF\Hal\Entity;
```

Comment:
Of course, they're in the same namespace, so no change is necessary here. My bad!